frontend/javascripts/test/puppeteer/dtype_datasets_rendering.screenshot.ts

```diff
@@ -237,11 +237,12 @@ describe("DType Dataset Rendering", () => {
               actions.push(updateLayerSettingAction(layerName, "intensityRange", intensityRange));
             }
 
-            await page.evaluate((actions) => {
-              return window.webknossos.apiReady().then(() => {
+            await page.evaluate(async (actions) => {
+              await window.webknossos.apiReady().then(async () => {
```

Comment:
The `then` callback actually gets `api` as an argument, so you could use that instead of `window.webknossos.DEV.api`.